lifeguards_1801_bronze.java

```diff
@@ -0,0 +1,54 @@
+/*
+Bill Zheng
+Time: 20 min
+USACO Test Cases: **********
+1-10 Difficulty: 3, harder to come up with a solution but implementation and writing was very easy.
+Reflection: The only hard part was coming up with the solution and then translating the hours to an array correctly because
+	it's not perfectly in line. Got all test cases in the first try.
+ */
+
+import java.io.*;
+import java.util.*;
+
+public class lifeguards_1801_bronze {
+	public static void main(String[] args) throws IOException {
+		Scanner scan = new Scanner (new File ("lifeguards.in"));
+//--------------------------------------------------------------------------------------------------------------------------
+		int N = scan.nextInt();
+		int[][] shifts = new int[N][2]; //inputs lifeguards shifts into a 2D array/matrix
+		for (int i = 0; i < N; i++) {
+			shifts[i][0] = scan.nextInt();
+			shifts[i][1] = scan.nextInt();
+		}
+//--------------------------------------------------------------------------------------------------------------------------
+		int[] time = new int[1001]; //timeline array to track hours

```

Comment:
This is a USACO *bronze* only hacky solution. It does not scale to higher levels or practical problems. Try work out alternate solution.